sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/Expression.scala

```diff
@@ -499,6 +489,8 @@ abstract class BinaryOperator extends BinaryExpression with ExpectsInputTypes {
 
   def symbol: String
 
+  def sqlOperator: String = symbol
```

Comment:
Oh, although I added the method, I forgot to use it in `And` and `Or` in the revision you commented at first. Just fixed that in my last commit.